visualization_scripts/environment.yml

```diff
@@ -0,0 +1,101 @@
+name: py2cyto
+channels:
+  - defaults
+dependencies:
+  - backcall=0.1.0=py35_0
+  - blas=1.0=mkl
+  - bleach=3.1.0=py_0
+  - ca-certificates=2019.10.16=0
+  - certifi=2018.8.24=py35_1
+  - colorama=0.4.1=py_0
+  - decorator=4.4.1=py_0
+  - defusedxml=0.6.0=py_0
+  - entrypoints=0.2.3=py35_2
+  - icc_rt=2019.0.0=h0cc432a_1
+  - icu=58.2=ha66f8fd_1
+  - intel-openmp=2019.4=245
+  - ipykernel=4.10.0=py35_0
+  - ipython=6.5.0=py35_0
+  - ipython_genutils=0.2.0=py35ha709e79_0
+  - ipywidgets=7.4.1=py35_0
+  - jedi=0.12.1=py35_0
+  - jinja2=2.10.3=py_0
+  - jpeg=9b=hb83a4c4_2
+  - jsonschema=2.6.0=py35h27d56d3_0
+  - jupyter=1.0.0=py35_7
+  - jupyter_client=5.3.3=py_0
+  - jupyter_console=5.2.0=py35_1
+  - jupyter_core=4.5.0=py_0
+  - libpng=1.6.37=h2a8f88b_0
+  - libsodium=1.0.16=h9d3ae62_0
+  - m2w64-gcc-libgfortran=5.3.0=6
+  - m2w64-gcc-libs=5.3.0=7
+  - m2w64-gcc-libs-core=5.3.0=7
+  - m2w64-gmp=6.1.0=2
+  - m2w64-libwinpthread-git=5.0.0.4634.697f757=2
+  - markupsafe=1.0=py35hfa6e2cd_1
+  - mistune=0.8.3=py35hfa6e2cd_1
+  - mkl=2018.0.3=1
+  - mkl_fft=1.0.6=py35hdbbee80_0
+  - mkl_random=1.0.1=py35h77b88f5_1
+  - msys2-conda-epoch=20160418=1
+  - nbconvert=5.5.0=py_0
+  - nbformat=4.4.0=py35h908c9d9_0
+  - notebook=5.6.0=py35_0
+  - numpy-base=1.15.2=py35h8128ebf_0
+  - openssl=1.1.1d=he774522_3
+  - pandoc=2.2.3.2=0
+  - pandocfilters=1.4.2=py35_1
+  - parso=0.5.1=py_0
+  - pickleshare=0.7.4=py35h2f9f535_0
+  - pip=10.0.1=py35_0
+  - prometheus_client=0.7.1=py_0
+  - prompt_toolkit=1.0.15=py35h89c7cb4_0
+  - pygments=2.4.2=py_0
+  - pyqt=5.9.2=py35h6538335_2
+  - python=3.5.4=h1357f44_23
+  - python-dateutil=2.8.1=py_0
+  - pywinpty=0.5.4=py35_0
+  - pyzmq=17.1.2=py35hfa6e2cd_0
+  - qt=5.9.7=vc14h73c81de_0
+  - qtconsole=4.6.0=py_0
+  - scipy=1.1.0=py35h4f6bf74_1
+  - send2trash=1.5.0=py35_0
+  - setuptools=40.2.0=py35_0
+  - simplegeneric=0.8.1=py35_2
+  - sip=4.19.8=py35h6538335_0
+  - sqlite=3.30.1=he774522_0
+  - tbb=2019.4=h74a9793_0
+  - tbb4py=2018.0.5=py35he980bc4_0
+  - terminado=0.8.1=py35_1
+  - testpath=0.4.4=py_0
+  - tornado=5.1.1=py35hfa6e2cd_0
+  - traitlets=4.3.2=py35h09b975b_0
+  - vc=14.1=h0510ff6_4
+  - vs2015_runtime=14.16.27012=hf0eaf9b_0
+  - wcwidth=0.1.7=py35h6e80d8a_0
+  - webencodings=0.5.1=py35_1
+  - wheel=0.31.1=py35_0
+  - widgetsnbextension=3.4.1=py35_0
+  - win_unicode_console=0.5=py35h56988b5_0
+  - wincertstore=0.2=py35hfebbdb8_0
+  - winpty=0.4.3=4
+  - zeromq=4.2.5=he025d50_1
+  - zlib=1.2.11=h62dcd97_3
+  - pip:
+    - chardet==3.0.4
+    - idna==2.8
+    - networkx==2.4
+    - numpy==1.17.4
```

Comment:
We should install as much as possible all within conda so that conda can resolve the correct versions.  The first time I created the environment, numpy 1.15.2 was installed through conda.  This pip section would then have uninstalled that version to replace it with 1.17.4, which may or may not be compatible with the other conda-installed packages.